src/azure-cli/azure/cli/command_modules/apim/custom.py

```diff
@@ -53,13 +54,26 @@ def _get_vs_fullpath(versionSetId):
     return fullpath
 
 
+def _get_subscription_key_parameter_names(subscription_key_query_param_name=None, subscription_key_header_name=None):
+    names = None
+    if subscription_key_query_param_name is not None and subscription_key_header_name is not None:
+        names = SubscriptionKeyParameterNamesContract(
+            header=subscription_key_header_name,
+            query=subscription_key_query_param_name
+        )
+    elif subscription_key_query_param_name is not None or subscription_key_header_name is not None:
+        raise CLIError(
```

Comment:
Please use a more specific error type `from azure.cli.core.azclierror import RequiredArgumentMissingError` instead of the generic error `CLIError`.
BTW, please check other errors you used to see if they also need update error type.